src/app-profile.ts

```diff
@@ -29,7 +29,7 @@ export interface AppProfileOptions {
    * value is required when creating the app profile and optional when setting
    * the metadata.
    */
-  routing?: 'any' | Cluster;
+  routing?: 'any' | Cluster | Array<Cluster>;
```

Comment:
The Java client uses a set instead of an array. Is there a compelling reason to have an array here? If no, can we switch to a set?

Reply:
Sounds like a good plan. I can change to a set. The only reason we would want to use an array is if we want for some reason for the order to be preserved every time we make a call to get the clusters in an app profile. I can't see why we would care about that though so a set is fine.

Reply:
The only consideration is that in the response object we get an array so maybe we want an array input for consistency, but probably it doesn't matter.